extensions/src/AWSSDK.Extensions.Bedrock.MEAI/BedrockChatClient.cs

```diff
@@ -794,84 +865,224 @@ private static Document ToDocument(JsonElement json)
         }
     }
 
-    /// <summary>Creates an <see cref="ToolConfiguration"/> from the specified options.</summary>
+    /// <summary>Creates a <see cref="ToolConfiguration"/> from the specified options.</summary>
+    /// <remarks>
```

Comment:
nit: remove these lines

```
/// <remarks>
    /// When ResponseFormat is specified, creates a synthetic tool to enforce structured output.
    /// This conflicts with user-provided tools as Bedrock only supports a single ToolChoice value.
    /// </remarks>
```

since we have in the other function docs